<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="59194" started-at="2017-03-30T12:33:04Z" finished-at="2017-03-30T12:34:03Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="59194" started-at="2017-03-30T12:33:04Z" finished-at="2017-03-30T12:34:03Z">
      <class name="appium.ios.OrderFlowTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:appium.ios.OrderFlowTest@153f5a29]" name="setUp" is-config="true" duration-ms="47188" started-at="2017-03-30T14:33:04Z" finished-at="2017-03-30T14:33:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testAndroid()[pri:0, instance:appium.ios.OrderFlowTest@153f5a29]" name="testAndroid" duration-ms="10264" started-at="2017-03-30T14:33:51Z" finished-at="2017-03-30T14:34:01Z">
          <exception class="java.lang.NoSuchMethodError">
            <message>
              <![CDATA[com.google.common.base.Preconditions.checkArgument(ZLjava/lang/String;J)V]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoSuchMethodError: com.google.common.base.Preconditions.checkArgument(ZLjava/lang/String;J)V
	at io.appium.java_client.pagefactory.TimeOutDuration.setTime(TimeOutDuration.java:54)
	at io.appium.java_client.pagefactory.TimeOutDuration.setTime(TimeOutDuration.java:59)
	at io.appium.java_client.pagefactory.TimeOutDuration.<init>(TimeOutDuration.java:37)
	at io.appium.java_client.pagefactory.AppiumFieldDecorator.<init>(AppiumFieldDecorator.java:89)
	at appium.ios.OrderFlowTest.testAndroid(OrderFlowTest.java:113)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoSuchMethodError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAndroid -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:appium.ios.OrderFlowTest@153f5a29]" name="tearDown" is-config="true" duration-ms="1718" started-at="2017-03-30T14:34:01Z" finished-at="2017-03-30T14:34:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- appium.ios.OrderFlowTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
